06-building-a-program-start-to-finish/solving-the-problem/output/inverted-index.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+
```

Comment:
i am calling some static methods in my InvertedIndexUtility class and es-lint is saying InvertedIndexUtility is not defined. So i had to ignore it.